### ISSUE TITLE
[Object Detection][Feature Request] Allow more than one eval_spec

### ISSUE THREAD

**Issue by netanel-s** [tensorflow/models#5982]
- **What is the top-level directory of the model you are using**: object_detection
- **Have I written custom code**: no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.10.
- **Bazel version (if compiling from source)**: -
- **CUDA/cuDNN version**: CUDA 9
- **GPU model and memory**: 1080Ti, 11GB
- **Exact command to reproduce**: Evaluating on more than one dataset/metrics_set while training 

### Describe the problem
In [training and evaluation](https://github.com/tensorflow/models/blob/master/research/object_detection/model_main.py#L105) case of `model_main` you restrict to a single evaluation spec. 
It would be very appreciated if could support more than one evaluation spec.
This would mean another evaluation sets (e.g. validation sets of several datasets), with corresponding protocols (metrics_set).

Thanks in advance.

**Comment by ppwwyyxx** [comment]
[My implementation](https://github.com/tensorpack/tensorpack/tree/master/examples/FasterRCNN) of Faster R-CNN and Mask R-CNN supports evaluating on arbitrary number of validation datasets.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.

**Comment by netanel-s** [comment]
It is still an open issue - valid feature request.
